### PR TITLE
OSDOCS#11006: Migrating HCP docs for IBM power from ACM to OCP

### PR DESCRIPTION
**Pull request by xenolinux**
This PR is a part of HCP content migration and modularization. This content is only being moved from RHACM to OCP and not changed. This content has been SME and peer reviewed. This migration PR contains the QE approval 

Version(s): 4.17 and main

Issue: https://issues.redhat.com/browse/OSDOCS-11006 

Link to docs preview:
- [Deploying hosted control planes on IBM Power](https://81599--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-deploy/hcp-deploy-ibmpower.html )
- [Prerequisites to configure hosted control planes on IBM Power](https://81599--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-deploy/hcp-deploy-ibmpower.html#hcp-ibmpower-prereqs_hcp-deploy-ibmpower)
- [IBM Power infrastructure requirements](https://81599--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-deploy/hcp-deploy-ibmpower.html#hcp-ibmpower-infra-reqs_hcp-deploy-ibmpower)
- [DNS configuration for hosted control planes on IBM Power](https://81599--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-deploy/hcp-deploy-ibmpower.html#hcp-ibmpower-dns_hcp-deploy-ibmpower)
- [Creating a hosted cluster on bare metal](https://81599--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-deploy/hcp-deploy-ibmpower.html#hcp-bm-hc_hcp-deploy-ibmpower)
- [Creating an InfraEnv resource for hosted control planes on IBM Power](https://81599--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-manage/hcp-manage-ibmpower.html#hcp-ibmpower-infraenv_hcp-manage-ibmpower)
- [Adding IBM Power agents to the InfraEnv resource](https://81599--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-manage/hcp-manage-ibmpower.html#hcp-ibmpower-add-agents_hcp-manage-ibmpower)
- [Scaling the NodePool object for a hosted cluster on IBM Power](https://81599--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-manage/hcp-manage-ibmpower.html#hcp-ibmpower-scale-np_hcp-manage-ibmpower)
- [Third bullet -> Additional resources -> Prerequisites to configure hosted control planes on IBMz](https://81599--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-deploy/hcp-deploy-ibmz.html#hcp-ibmz-prereqs_hcp-deploy-ibmz)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/hcp-bm-hc.adoc [hunk 1]
```diff
@@ -63,11 +64,11 @@ $ hcp create cluster agent \
 $ oc -n <hosted_control_plane_namespace> get pods
 ----
 +
-.Example
+.Example output
 ----
```
Comment (bhardesty) [inline modules/hcp-bm-hc.adoc:68]:
```suggestion
[source,terminal]
----
```

File: modules/hcp-ibmpower-scale-np.adoc [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+----
+
+. Run the following command to check the Cluster Operator status:
++
+[source,terminal]
+----
+$ oc --kubeconfig <hosted_cluster_name>.kubeconfig get clusteroperators
+----
++
+For each component of your cluster, the output shows the following Cluster Operator statuses:
```
Comment (bhardesty) [inline modules/hcp-ibmpower-scale-np.adoc:117]: This bulleted list is not formatted correctly in the output. I think you just need to add a line break before the first bullet point.

**Review by bhardesty — COMMENTED** [review]
Looks good! Since this was previously reviewed and published, I focused on the output (to make sure the content that was ported over is formatted correctly). I just noticed a couple tiny things you might want to fix. Otherwise looks great!

File: modules/hcp-ibmpower-scale-np.adoc (hunk 2 above)
Comment (xenolinux) [reply modules/hcp-ibmpower-scale-np.adoc:117]: Fixed

File: modules/hcp-bm-hc.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/hcp-bm-hc.adoc:69]: Fixed

**Pull request merged**